backends-common/elasticsearch/src/test/java/org/apache/james/backends/es/ElasticSearchConfigurationTest.java

```diff
@@ -66,6 +66,30 @@ public void getNbReplicaShouldReturnDefaultValueWhenMissing() throws Configurati
             .isEqualTo(ElasticSearchConfiguration.DEFAULT_NB_REPLICA);
     }
 
+    @Test
+    public void getWaitForActiveShardsShouldReturnConfiguredValue() throws ConfigurationException {
+        PropertiesConfiguration configuration = new PropertiesConfiguration();
+        int value = 36;
```

Comment:
s/value/configuredWaitForActiveShards ?

Reply:
I love short names in a less than 10 lines method, and also it's matching the global pattern of the test class.